R/update-news.R

```diff
@@ -125,3 +136,21 @@ edit_cran_comments <- function() {
   local_options(usethis.quiet = TRUE)
   edit_file("cran-comments.md")
 }
+
+is_merge_commit <- function(message) {
+  grepl("^Merge pull request #[0-9]* from", message)
```

Comment:
Make use of rematch2?

```suggestion
  grepl("^Merge pull request #([0-9]*) from", message)
```